[2024-12-17T16:41:13.376+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T16:41:13.382+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 manual__2024-12-17T16:41:10.982708+00:00 [queued]>
[2024-12-17T16:41:13.386+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 manual__2024-12-17T16:41:10.982708+00:00 [queued]>
[2024-12-17T16:41:13.386+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-17T16:41:13.390+0000] {taskinstance.py:2889} INFO - Executing <Task(TriggerDagRunOperator): trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2> on 2024-12-17 16:41:10.982708+00:00
[2024-12-17T16:41:13.393+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1649) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-17T16:41:13.394+0000] {standard_task_runner.py:72} INFO - Started process 1653 to run task
[2024-12-17T16:41:13.393+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'AF_TEST1', 'trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2', 'manual__2024-12-17T16:41:10.982708+00:00', '--job-id', '2143', '--raw', '--subdir', 'DAGS_FOLDER/script/AF_TEST1.py', '--cfg-path', '/tmp/tmp1icbfnr_']
[2024-12-17T16:41:13.394+0000] {standard_task_runner.py:105} INFO - Job 2143: Subtask trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2
[2024-12-17T16:41:13.410+0000] {task_command.py:467} INFO - Running <TaskInstance: AF_TEST1.trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2 manual__2024-12-17T16:41:10.982708+00:00 [running]> on host 2dca8d069141
[2024-12-17T16:41:13.436+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test-framework' AIRFLOW_CTX_DAG_ID='AF_TEST1' AIRFLOW_CTX_TASK_ID='trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T16:41:10.982708+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-17T16:41:10.982708+00:00'
[2024-12-17T16:41:13.436+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T16:41:15.349+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:175 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.349+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:222 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.350+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1151 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.351+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1248 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.351+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1271 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.351+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1406 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.352+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1439 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.352+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1495 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.352+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1683 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.353+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1750 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.353+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2025 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.353+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2110 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.354+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2407 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.354+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2468 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.356+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:3987 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.356+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4096 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.356+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4127 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.357+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4186 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.357+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4225 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.357+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4275 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.358+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5209 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.358+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5265 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.359+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5302 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.359+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5358 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.360+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6437 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.360+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6497 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.361+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6538 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.361+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.362+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7023 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.362+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7096 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.362+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7267 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.362+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7333 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.363+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7378 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.363+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7440 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.364+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8222 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.364+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8282 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.364+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8322 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.365+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8368 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.366+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:9475 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.366+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:9531 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.367+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.367+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.368+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10625 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.368+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10680 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.369+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10923 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.369+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10979 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.369+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11016 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.369+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11072 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.370+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11231 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.370+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11282 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.370+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11420 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.371+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11477 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.371+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12300 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.372+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12383 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.372+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12439 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.372+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12524 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.373+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13019 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.373+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13090 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.373+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13139 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.374+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13200 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.374+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13886 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.375+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13941 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.375+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13977 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.375+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14026 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.376+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14318 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.376+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14388 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.376+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14436 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.376+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14492 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.377+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14744 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.377+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14810 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.378+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15415 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.378+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15482 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.379+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15527 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.379+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15578 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.379+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15897 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.380+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15953 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.380+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16617 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.381+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16674 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.381+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16712 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.381+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16780 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.381+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.382+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16988 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.382+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17025 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.382+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17090 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.382+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17228 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.383+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17284 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.386+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20234 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.386+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20289 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.387+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20862 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.387+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20921 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.387+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20960 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.387+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21014 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.389+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21316 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.389+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21377 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.389+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21418 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.390+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21478 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.390+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21612 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.390+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21668 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.391+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22155 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.391+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22218 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.392+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22559 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.392+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22626 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.392+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22671 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.393+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22722 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.393+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23061 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.394+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23128 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.394+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23173 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.394+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23224 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.394+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23511 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.395+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23567 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.395+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23968 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.396+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:24018 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.397+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25343 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.397+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25398 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.398+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25434 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.398+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25485 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.399+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:26723 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.400+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:26779 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.400+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27121 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.400+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27177 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.401+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27461 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.401+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27517 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.402+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28053 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.402+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28109 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.403+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28536 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.403+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28592 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.404+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.404+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28982 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.406+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:30727 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.406+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:30783 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.409+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31296 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.410+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31344 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.412+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31617 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.413+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31673 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.414+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32120 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.414+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32176 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.415+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32339 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.415+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32395 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.415+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32432 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.416+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32487 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.417+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33502 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.417+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33558 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.418+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33869 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.418+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33928 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.418+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33967 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.419+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34021 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.420+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34739 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.420+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34806 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.420+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34852 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.420+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34908 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.423+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:36915 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.423+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:36971 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.424+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37417 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.424+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37473 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.424+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37845 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.425+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37901 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.425+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38208 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.425+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38264 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.426+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38301 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.426+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38351 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.426+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38690 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.427+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38758 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.427+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39141 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.428+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39192 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.428+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39419 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.428+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39487 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.429+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39683 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.429+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39738 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.430+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:40019 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.430+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:40084 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.432+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41138 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.432+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41196 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.432+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41359 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.433+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41415 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.433+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41452 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.433+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41507 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.434+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41875 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.434+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41947 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.438+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42441 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.438+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42497 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.438+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42534 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.439+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42590 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.439+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42627 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.439+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42692 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.440+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43020 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.440+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43079 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.440+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43118 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.440+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43168 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.441+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43514 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.441+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43573 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.441+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43612 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.442+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43666 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.442+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43989 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.442+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44045 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.443+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44386 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.443+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44442 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.445+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45003 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.445+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45058 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.446+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45444 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.446+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.447+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46042 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.447+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46098 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.448+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46702 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.448+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46758 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.449+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47000 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.449+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47050 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.450+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47648 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.450+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47704 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.451+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48113 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.451+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48190 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.452+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48244 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.452+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48321 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.454+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.454+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49555 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.455+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49932 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.455+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50002 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.455+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50052 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.455+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50107 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.456+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50379 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.456+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50449 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.456+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50499 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.457+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50554 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.457+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50590 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.457+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.458+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50710 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.458+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50770 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.458+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51046 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.459+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51116 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.459+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51166 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.459+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51226 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.460+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51485 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.460+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51540 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.460+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51787 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.460+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51851 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.461+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51895 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.461+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51958 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.461+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52212 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.462+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52275 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.462+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52463 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.462+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.463+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53040 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.463+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53111 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.464+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53334 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.464+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53407 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.465+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53961 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.465+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54059 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.466+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54322 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.466+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54398 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.467+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55022 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.467+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55078 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.467+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55310 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.468+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55365 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.469+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56196 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.469+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56248 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.469+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56508 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.470+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56564 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.470+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57087 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.471+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57147 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.471+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57187 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.471+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57237 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.472+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57560 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.472+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57620 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.472+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.473+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57710 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.474+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59106 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.475+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59162 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.475+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59316 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.475+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59396 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.476+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59452 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.476+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59535 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.476+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59594 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.476+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59677 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.477+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59733 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.477+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59818 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.478+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60639 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.478+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60722 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.478+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60778 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.479+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60863 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.479+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61167 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.480+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61250 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.480+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61306 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.480+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61387 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.481+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61696 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.481+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61752 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.481+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61954 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.482+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61992 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.482+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:62660 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.483+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:62715 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.484+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:63882 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.484+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:63946 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.485+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64582 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.486+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64674 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.486+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64940 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.486+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65014 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.487+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65117 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.487+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65199 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.487+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65385 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.488+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65441 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.488+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65874 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.488+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65950 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.489+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66003 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.489+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66087 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.490+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66686 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.490+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66779 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.490+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66918 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.491+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66964 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.492+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:67735 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.492+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:67791 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.492+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68043 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.493+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68097 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.493+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68532 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.494+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68588 SyntaxWarning: invalid escape sequence '\d'
[2024-12-17T16:41:15.750+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T16:41:15.750+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=AF_TEST1, task_id=trigger_PRCS_TEST_FRAMEWORK1_prcsgrp_PRCS_GRP_TEST_FRAMEWORK1_prcsgrpprir_1_prcsprir_2, run_id=manual__2024-12-17T16:41:10.982708+00:00, execution_date=20241217T164110, start_date=20241217T164113, end_date=20241217T164115
[2024-12-17T16:41:15.776+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T16:41:15.787+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-17T16:41:15.787+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
